<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workerwindow</class>
 <widget class="QWidget" name="workerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>141</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_page_mainwindow">
      <property name="text">
       <string>主页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_data_find">
      <property name="text">
       <string>个人信息查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_dept_find">
      <property name="text">
       <string>所属部门</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_salary_find">
      <property name="text">
       <string>薪资查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_worker_go_find">
      <property name="text">
       <string>出勤信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_leave_find">
      <property name="text">
       <string>请假申请</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_leave_look">
      <property name="text">
       <string>查看请假</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_context_find">
      <property name="text">
       <string>查看公告</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_changed_password">
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_exit">
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>727</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>8</number>
   </property>
   <widget class="QWidget" name="pagenoe">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>41</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>工号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_id">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_account">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_password">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>名字：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_sex">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_iphone">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>99</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>电话：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>部门编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_deptid">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_email">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_address">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>170</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_dates">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_identify">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_workers_state">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>240</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>电子邮件：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>在职状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>入职日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lineEdit_workers_remark">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>581</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagetwo">
    <widget class="QTableView" name="tableView_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>711</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_update_data">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rest">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagethree">
    <widget class="QTableView" name="tableView_dept">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>741</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dept_back">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>450</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagefour">
    <widget class="QTableView" name="tableView_salary">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>701</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagefive">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>681</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>450</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagesix">
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leave_id">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leave_workername">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leave_beginday">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>开始时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>结束时间：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leave_endday">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>请假事由：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="leave_description">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>451</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_leave_sure">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_leave_rest">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagesenven">
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageeight">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>591</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪欧行繁</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>欢迎进入员工管理系统</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagepwd">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>原密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认新密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主页</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>员工管理系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
